icu4c/source/test/cintltst/cbiapts.c

```diff
@@ -515,6 +517,79 @@ static void TestBreakIteratorSafeClone(void)
         ubrk_close(someIterators[i]);
     }
 }
+
+static void TestBreakIteratorClone(void)
+{
+    UChar text[51];     /* Keep this odd to test for 64-bit memory alignment */
+                        /*  NOTE:  This doesn't reliably force mis-alignment of following items. */
+    UBreakIterator * someIterators [CLONETEST_ITERATOR_COUNT];
+    UBreakIterator * someClonedIterators [CLONETEST_ITERATOR_COUNT];
+
+    UErrorCode status = U_ZERO_ERROR;
+    int32_t start,pos;
+    int32_t i;
+
+    /*Testing ubrk_clone */
+
+    /* Note:  the adjacent "" are concatenating strings, not adding a \" to the
+       string, which is probably what whoever wrote this intended.  Don't fix,
+       because it would throw off the hard coded break positions in the following
+       tests. */
+    u_uastrcpy(text, "He's from Africa. ""Mr. Livingston, I presume?"" Yeah");
+
+    /* US & Thai - rule-based & dictionary based */
+    someIterators[0] = ubrk_open(UBRK_WORD, "en_US", text, u_strlen(text), &status);
+    if(!someIterators[0] || U_FAILURE(status)) {
+      log_data_err("Couldn't open en_US word break iterator - %s\n", u_errorName(status));
+      return;
+    }
+
+    someIterators[1] = ubrk_open(UBRK_WORD, "th_TH", text, u_strlen(text), &status);
+    if(!someIterators[1] || U_FAILURE(status)) {
+      log_data_err("Couldn't open th_TH word break iterator - %s\n", u_errorName(status));
+      return;
+    }
+
+    /* test each type of iterator */
+    for (i = 0; i < CLONETEST_ITERATOR_COUNT; i++)
+    {
+
+        /* Check the various error & informational states */
+
+        /* Null status - just returns NULL */
+        if (NULL != ubrk_clone(someIterators[i],  NULL))
```

Comment:
use nullptr for new code here and below please

Reply:
You can't use nullptr in a .c file.